### PR TITLE
Add a configuration for dual bank for DISCO_F769NI

### PR DESCRIPTION
**Pull request by jarlamsa**
By default, the board is configured to use single bank.
To enable dual bank, one has to use ST-Link tool to enable
it first in the device.

### CODE REVIEW

**Comment by jarlamsa** [comment]
@debdeep-arm could you verify the new configuration? It should be similar that you are using.

File: configs/internal_flash_dual_bank_no_rot.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            "mbed-bootloader.application-start-address": "(MBED_CONF_UPDATE_CLIENT_APPLICATION_DETAILS + MBED_BOOTLOADER_ACTIVE_HEADER_REGION_SIZE)",
+            "mbed-bootloader.max-application-size"     : "(MBED_ROM_START + MBED_BOOTLOADER_FLASH_BANK_SIZE - MBED_CONF_MBED_BOOTLOADER_APPLICATION_START_ADDRESS)",
+            "update-client.storage-address"            : "(MBED_ROM_START + MBED_BOOTLOADER_FLASH_BANK_SIZE + KVSTORE_SIZE)",
+            "update-client.storage-size"               : "(MBED_BOOTLOADER_FLASH_BANK_SIZE - KVSTORE_SIZE)",
+            "update-client.storage-locations"          : 1,
+            "target.extra_labels_remove"               : [ "PSA" ],
+            "update-client.storage-page"               : 1
+        },
+        "DISCO_F769NI": {
+            "target.flash_dual_bank"                   : 1,
```
Comment (teetak01) [inline configs/internal_flash_dual_bank_no_rot.json:106]: considering the file name, this config could be on target definitions. Similar thing may be needed for other F7-series boards also.

Comment (jarlamsa) [reply configs/internal_flash_dual_bank_no_rot.json:106]: `target.flash_dual_bank` is specific only for some F76X-boards. At the moment I would only keep it in the device-specific override.

Comment (jeromecoutant) [reply configs/internal_flash_dual_bank_no_rot.json:106]: yes
https://os.mbed.com/teams/ST/wiki/How-to-enable-flash-dual-bank

**Comment by jarlamsa** [comment]
Build failed for `UBLOX_EVK_ODIN_W2` and `LPC55S69_NS` as Odin has been removed from mbed-os master and LPC55S69_NS has been temporarily disabled.

**Pull request merged**